skill-manager/.local.env

```diff
@@ -3,4 +3,4 @@ MONGO_INITDB_ROOT_PASSWORD=mongo-local-pw
 REDIS_USER=default
 REDIS_PASSWORD=redis-local-pw
 CLIENT_SECRET=skill-manager-local-pw
-EVALUATOR_API_URL=
```

Comment:
Können wir das hier standardmäßig ändern auf die Production API (also wie auch in den anderen .env Dateien)?
Also `https://square.ukp-lab.de/api/evaluator` müsste es ja sein.